### PR TITLE
Fix up mariadb.

### PR DESCRIPTION
**Pull request by dlorenc**
I tweaked the mariadb package to more closely match the APKBUILD from Alpine. This involved:
* passing a lot more build options which required more dependencies during the build
* splitting up the package (this trims it significantly)
* adding libaio (for the first bullet)
* adding the docker-entrypoint

Signed-off-by: Dan Lorenc <dlorenc@chainguard.dev>

### CODE REVIEW

**Comment by dlorenc** [comment]
The new package is 63MB vs. 462MB. I verified it runs in an image!

**Comment by dlorenc** [comment]
D'oh - I need to split this up: https://github.com/wolfi-dev/os/pull/463

**Review by luhring — DISMISSED** [review]

File: mariadb.yaml [hunk 1]
```diff
@@ -25,22 +25,175 @@ environment:
...
+        -DSKIP_TESTS=ON
   - name: "Install"
     runs: |
       DESTDIR="${{targets.destdir}}" ninja install
   - name: "Remove extras"
     runs: |
-      rm -rf "${{targets.destdir}}/usr/local/mysql/sql-bench/"
-      rm -rf "${{targets.destdir}}/usr/local/mysql/mysql-test/"
+      rm -rf "${{targets.destdir}}"/usr/local/mysql/sql-bench/"
+      rm -rf "${{targets.destdir}}"/usr/local/mysql/mysql-test/"
```
Comment (jdolitsky) [inline mariadb.yaml:113]: bad quotes here

**Review by jdolitsky — CHANGES_REQUESTED** [review]

File: mariadb.yaml (hunk 1 above)
Comment (dlorenc) [reply mariadb.yaml]: Fixed!

**Review by jdolitsky — APPROVED** [review]

**Review by kaniini — APPROVED** [review]

**Pull request merged**